src/Microsoft.ML.Data/Transforms/ColumnSelecting.cs

```diff
@@ -33,8 +33,32 @@
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
-    /// The ColumnSelectingEstimator supports selection of specified columns to keep from a given input.
+    /// Keeps or drops selected columns from an <see cref="IDataView"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input columns data type | Any |
+    /// | Output column data type | When not dropped, the data type is unchanged |
+    ///
+    /// The resulting [ColumnSelectingTransformer](<xref:Microsoft.ML.Transforms.ColumnSelectingTransformer>)
+    /// operates on the schema of a given [IDataView](<xref:Microsoft.ML.IDataView>) dropping or keeping selected columns from the schema.
```

Comment:
>[IDataView](<xref:Microsoft.ML.IDataView>) [](start = 42, length = 42)

ditto here an everywhere. 
[anchor](<xref:UID>) is incorrect b/c of extra <>

If the anchor text is the same as the class/type name, you don't really need to use [anchor](xref:UID) pattern. You can save some time by using autolinks: <xref:UID>